ccd-definition/CaseEvent.json

```diff
@@ -298,6 +298,19 @@
     "SecurityClassification": "Public",
     "ShowSummary": "N",
     "ShowEventNotes": "N",
-    "EndButtonLabel": "Save and continue"    
+    "EndButtonLabel": "Save and continue"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "Shared_Storage_DRAFTType",
+    "ID": "sendToGatekeeper",
+    "Name": "Send to gatekeeper",
+    "Description": "Send email to gatekeeper",
+    "DisplayOrder": 1,
+    "PreConditionState(s)": "Submitted",
+    "PostConditionState": "Submitted",
+    "SecurityClassification": "Public",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/about-to-submit/notify-gatekeeper",
+    "EndButtonLabel": "Save and continue"
```

Comment:
Can you double check with business the label? `Save and continue` looks wired for that action. I expect something like `Submit` to `Send`.

Reply:
i already checked with Olivia yesterday, thats what she wanted